kpTranslator/controllers/query_controller.py

```diff
@@ -15,4 +19,30 @@ def query(request_body):  # noqa: E501
 
     :rtype: Message
     """
-    return 'do some magic!'
+    # Parse request
+    request = Query.from_dict(request_body)
+
+    try:
+        # Get query graph nodes
+        query_graph = request.message.query_graph
+        qnodes = query_graph.nodes
+    except AttributeError as e:
+        return 'Query request is invalid!\n{}'.format(e)
```

Comment:
If you change this to:
```
return 'Query request is invalid!\n{}'.format(e), 422
```
then it will return a 422 response.  You'll also need to add a 422 to the yaml kpTranslator//openapi/openapi.yaml so that it shows up as an expected return.